server-config.json

```diff
@@ -12,5 +12,5 @@
     "feePercentage": "0",
     "disableSponsoredTx": true,
     "logLevel": 1,
-    "workdir": "/Users/borja/iovlabs/projects/rif-relay/environment"
+    "workdir": "/home/user/workspace/enveloping/environment"
```

Comment:
1. doesn't look like a safe sort of path. if this is a configurable property then make it clear that it is.
2. could you please clarify why we do actually need this path?

Reply:
This is the path where the seeds for the relayWorker and relayManager are stored, it also has the pool of transactions. It needs to be a local address in the server. And its configurable thats why is in the config.json file

Reply:
1. sorry, what I meant was not-hardcoded basically. Like give it an empty string or a default path (which "/home/user/workspace/enveloping/environment" surely isn't as that is machine specific path).
2. should we then perhaps call it `seedDir` or something like that?